actions/add-review-url/src/AddReviewUrl.js

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const {graphql} = require('@octokit/graphql');

```

Comment:
Is `const {...}` different to `const ...`? If not, pick one.

Reply:
Yes. They are different. Here is a link https://stackoverflow.com/questions/38660022/curly-brackets-braces-in-node-require-statement which explains the difference.